manifests/init.pp

```diff
@@ -52,6 +54,7 @@
 #
 class foreman_proxy_content (
   String[1] $parent_fqdn = $foreman_proxy_content::params::parent_fqdn,
+  String[1] $foreman_fqdn = $foreman_proxy_content::params::foreman_fqdn,
```

Comment:
Another thought: instead of exposing the parameter use ```$::foreman_proxy::foreman_base_url```. That way you automatically get a working setup.